app/views/step_by_step_pages/new.html.erb

```diff
@@ -1,22 +1,17 @@
 <%
   links = [
     {
-      text: 'Step by step publisher',
+      text: "Step by step publisher",
       href: step_by_step_pages_path
     },
     {
-      text: 'Create new'
+      text: "Create a new step by step navigation"
```

Comment:
I think we can drop the word "navigation". What do you think?

Reply:
Sounds sensible and makes it consistent with the primary action on the summary page, will update